solutions/swb-reference/src/services/dataSetService.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CreateProvisionDatasetRequest } from '@aws/swb-app/lib/datasets/createProvisionDatasetRequest';
+import { DataSet } from '@aws/swb-app/lib/datasets/dataSet';
+import { DatasetPluginInterface } from '@aws/swb-app/lib/datasets/datasetPluginInterface';
+import { DataSetsStoragePluginInterface } from '@aws/swb-app/lib/datasets/dataSetsStoragePluginInterface';
+import { AuditService } from '@aws/workbench-core-audit';
+import {
+  DataSetMetadataPlugin,
+  DataSetService as WorkbenchDataSetService
+} from '@aws/workbench-core-datasets';
+import { LoggingService } from '@aws/workbench-core-logging';
+
+export class DataSetService implements DatasetPluginInterface {
+  public readonly storagePlugin: DataSetsStoragePluginInterface;
+  private _workbenchDataSetService: WorkbenchDataSetService;
+
+  public constructor(
+    storagePlugin: DataSetsStoragePluginInterface,
+    audit: AuditService,
+    log: LoggingService,
+    masterDbProvider: DataSetMetadataPlugin
+  ) {
+    this._workbenchDataSetService = new WorkbenchDataSetService(audit, log, masterDbProvider);
```

Comment:
Nitpick: `_workbenchDataSetService` => `_swbDataSetService` since this is for `swb`.  I'm not too tied to the rename though.

Reply:
This happened because `DataSetService` is also being imported from workbench core. You can see on line 15 that I had to alias the import with `  DataSetService as WorkbenchDataSetService`. `_swbDataSetService` would be what I'd call this file's class if we instantiated it.

Reply:
Thoughts on renaming it to `DataSetService as SwbDataSetService`?

Reply:
I think that would be misleading since we'd be importing from `workbench-core/dataset` but psuedo-namespacing it like it was actually from service work bench. None of this would be a breaking change so we can always change it later down the line if people get confused by referencing the `workbench-core` import as `WorkbenchDataSetService`. Thoughts?